### PR TITLE
All four exercise

### PR DESCRIPTION
**Pull request by kareraolivier**
## Correction of exercises

### CODE REVIEW

File: exercisethree.js
Comment (gsngdavid) [inline exercisethree.js:1]: Optional: put your if statements on one line since the body is made of single statement

File: exercisetwo.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  roman = roman.replace(/CM/g, "DCCCC");
+
+  for (let i = 0; i < roman.length; i++) {
+    result += romanNumbers[roman[i]];
+  }
+
+  return result;
+}
+
+console.log(romanToNumber("VII"));
```
Comment (gsngdavid) [inline exercisetwo.js]: You can remove console after testing

**Review by gsngdavid — CHANGES_REQUESTED** [review]
Well done

**Review by gsngdavid — APPROVED** [review]

File: exercisethree.js
Comment (kareraolivier) [reply exercisethree.js:1]: done

File: exercisetwo.js (hunk 1 above)
Comment (kareraolivier) [reply exercisetwo.js]: removed

**Pull request merged**